### DISCUSSION TITLE
How do I get a basic setup working?

### DISCUSSION THREAD

**Discussion by 1066JMay96** [libp2p/rust-libp2p#2870] [Q&A] [answered]
![picture](https://user-images.githubusercontent.com/56660247/188468654-d08c1f66-4b81-4284-996e-c3c59c681a26.jpg)
![picture2](https://user-images.githubusercontent.com/56660247/188468659-88dad6fe-fbeb-45e3-bbcd-9d11477e0695.jpg)

**Comment by jxs** [comment, marked as answer]
Hi!
Can you provide more info regarding your stack trace and build environment? It seems you are triggering an issue akin to https://github.com/rust-lang/rust/issues/100062. But could also be https://github.com/dtolnay/anyhow/issues/250. 
Does executing `cargo clean` and running with the latest stable Rust version (1.63) fix the issue?

**Comment by thomaseizinger** [comment]
Very likely you are running a too old version of Rust. Can you please post the output of:

- `rustc --version`
- `cargo --version`
